<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>993</width>
    <height>583</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label_trem1">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>70</y>
      <width>21</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { background: red;color:white}</string>
    </property>
    <property name="text">
     <string>T1</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_trilho1">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>30</y>
      <width>250</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { background: blue}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_trilho2">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>150</y>
      <width>250</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { background: blue}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_trilho3">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>30</y>
      <width>21</width>
      <height>137</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { background: blue}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_trilho4">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>30</y>
      <width>21</width>
      <height>137</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { background: blue}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_trilho4_2">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>30</y>
      <width>21</width>
      <height>137</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { background: blue}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_trilho1_2">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>30</y>
      <width>250</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { background: blue}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_trilho2_2">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>150</y>
      <width>250</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { background: blue}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_trem2">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>30</y>
      <width>21</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { background: green;color:white}</string>
    </property>
    <property name="text">
     <string>T2</string>
    </property>
   </widget>
   <widget class="QSlider" name="velTrem01">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>370</y>
      <width>160</width>
      <height>16</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="minimum">
     <number>0</number>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="value">
     <number>20</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="invertedAppearance">
     <bool>false</bool>
    </property>
    <property name="invertedControls">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QSlider" name="velTrem02">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>400</y>
      <width>160</width>
      <height>16</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="value">
     <number>20</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="velLabel01">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>370</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Trem 01</string>
    </property>
   </widget>
   <widget class="QLabel" name="velLabel02">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>400</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Trem 02</string>
    </property>
   </widget>
   <widget class="QLabel" name="central">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>340</y>
      <width>261</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Central de Velocidades:</string>
    </property>
   </widget>
   <widget class="QLabel" name="vel01">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>370</y>
      <width>81</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>100 ms</string>
    </property>
   </widget>
   <widget class="QLabel" name="vel02">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>400</y>
      <width>71</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>100 ms</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_trilho4_3">
    <property name="geometry">
     <rect>
      <x>195</x>
      <y>166</y>
      <width>21</width>
      <height>121</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { background: blue}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_trilho2_3">
    <property name="geometry">
     <rect>
      <x>215</x>
      <y>270</y>
      <width>250</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { background: blue}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_trilho4_4">
    <property name="geometry">
     <rect>
      <x>465</x>
      <y>166</y>
      <width>21</width>
      <height>121</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { background: blue}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="vel04">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>460</y>
      <width>71</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>100 ms</string>
    </property>
   </widget>
   <widget class="QSlider" name="velTrem04">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>460</y>
      <width>160</width>
      <height>16</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="value">
     <number>20</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="velLabel04">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>460</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Trem 04</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_trem4">
    <property name="geometry">
     <rect>
      <x>195</x>
      <y>190</y>
      <width>21</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { background: orange;color:white}</string>
    </property>
    <property name="text">
     <string>T4</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>335</x>
      <y>90</y>
      <width>16</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>150</y>
      <width>16</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>150</y>
      <width>16</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>340</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>x    -    y</string>
    </property>
   </widget>
   <widget class="QLabel" name="velLabel04_2">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>460</y>
      <width>81</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Trem 04 -&gt; </string>
    </property>
   </widget>
   <widget class="QLabel" name="velLabel01_2">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>370</y>
      <width>81</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Trem 01 -&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="velLabel02_2">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>400</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Trem 02 -&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="trem1x">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>370</y>
      <width>31</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="trem1y">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>370</y>
      <width>41</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="trem2y">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>400</y>
      <width>51</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="trem2x">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>400</y>
      <width>41</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="trem4y">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>460</y>
      <width>51</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="trem4x">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>460</y>
      <width>41</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_trilho1_3">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>30</y>
      <width>250</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { background: blue}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_trilho2_4">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>150</y>
      <width>250</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { background: blue}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_trilho4_5">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>30</y>
      <width>21</width>
      <height>137</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { background: blue}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_trem3">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>70</y>
      <width>21</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { background: purple;color:white}</string>
    </property>
    <property name="text">
     <string>T3</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>90</y>
      <width>16</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QSlider" name="velTrem03">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>430</y>
      <width>160</width>
      <height>16</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="value">
     <number>20</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="velLabel03">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>430</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Trem 03</string>
    </property>
   </widget>
   <widget class="QLabel" name="vel03">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>430</y>
      <width>71</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>100 ms</string>
    </property>
   </widget>
   <widget class="QLabel" name="trem3x">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>430</y>
      <width>41</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="velLabel04_3">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>430</y>
      <width>81</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Trem 03 -&gt; </string>
    </property>
   </widget>
   <widget class="QLabel" name="trem3y">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>430</y>
      <width>51</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_trilho2_5">
    <property name="geometry">
     <rect>
      <x>485</x>
      <y>270</y>
      <width>250</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { background: blue}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_trilho4_6">
    <property name="geometry">
     <rect>
      <x>735</x>
      <y>166</y>
      <width>21</width>
      <height>121</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { background: blue}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_trem5">
    <property name="geometry">
     <rect>
      <x>735</x>
      <y>190</y>
      <width>21</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { background: black;color:white}</string>
    </property>
    <property name="text">
     <string>T5</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>150</y>
      <width>16</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>150</y>
      <width>16</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>210</y>
      <width>16</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>6</string>
    </property>
   </widget>
   <widget class="QLabel" name="trem5x">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>490</y>
      <width>41</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="velLabel05">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>490</y>
      <width>81</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Trem 05 -&gt; </string>
    </property>
   </widget>
   <widget class="QLabel" name="trem5y">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>490</y>
      <width>51</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QSlider" name="velTrem05">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>490</y>
      <width>160</width>
      <height>16</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="value">
     <number>20</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="velLabel04_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>490</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Trem 05</string>
    </property>
   </widget>
   <widget class="QLabel" name="vel05">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>490</y>
      <width>71</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>100 ms</string>
    </property>
   </widget>
   <zorder>label_trilho2</zorder>
   <zorder>label_trilho1_2</zorder>
   <zorder>label_trilho2_2</zorder>
   <zorder>label_trilho4_4</zorder>
   <zorder>label_trilho4_3</zorder>
   <zorder>label_trilho4_2</zorder>
   <zorder>label_trilho4</zorder>
   <zorder>label_trilho1</zorder>
   <zorder>label_trilho3</zorder>
   <zorder>label_trem1</zorder>
   <zorder>label_trem2</zorder>
   <zorder>velTrem01</zorder>
   <zorder>velTrem02</zorder>
   <zorder>velLabel01</zorder>
   <zorder>velLabel02</zorder>
   <zorder>central</zorder>
   <zorder>vel01</zorder>
   <zorder>vel02</zorder>
   <zorder>label_trilho2_3</zorder>
   <zorder>vel04</zorder>
   <zorder>velTrem04</zorder>
   <zorder>velLabel04</zorder>
   <zorder>label_trem4</zorder>
   <zorder>label</zorder>
   <zorder>label_2</zorder>
   <zorder>label_3</zorder>
   <zorder>label_4</zorder>
   <zorder>velLabel04_2</zorder>
   <zorder>velLabel01_2</zorder>
   <zorder>velLabel02_2</zorder>
   <zorder>trem1x</zorder>
   <zorder>trem1y</zorder>
   <zorder>trem2y</zorder>
   <zorder>trem2x</zorder>
   <zorder>trem4y</zorder>
   <zorder>trem4x</zorder>
   <zorder>label_trilho1_3</zorder>
   <zorder>label_trilho2_4</zorder>
   <zorder>label_trilho4_5</zorder>
   <zorder>label_trem3</zorder>
   <zorder>label_5</zorder>
   <zorder>velTrem03</zorder>
   <zorder>velLabel03</zorder>
   <zorder>vel03</zorder>
   <zorder>trem3x</zorder>
   <zorder>velLabel04_3</zorder>
   <zorder>trem3y</zorder>
   <zorder>label_trilho2_5</zorder>
   <zorder>label_trilho4_6</zorder>
   <zorder>label_trem5</zorder>
   <zorder>label_6</zorder>
   <zorder>label_7</zorder>
   <zorder>label_8</zorder>
   <zorder>trem5x</zorder>
   <zorder>velLabel05</zorder>
   <zorder>trem5y</zorder>
   <zorder>velTrem05</zorder>
   <zorder>velLabel04_4</zorder>
   <zorder>vel05</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>993</width>
     <height>17</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
